benchmarks/src/main/scala/zio/redis/lists/BlPopBenchmarks.scala

```diff
@@ -0,0 +1,65 @@
+package zio.redis.lists
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+
+import zio.ZIO
+import zio.duration._
+import zio.redis.{ BenchmarkRuntime, blPop, rPush }
```

Comment:
I'd import everything regardless of the need, as long as it doesn't cause collisions.